package-lock.json

```diff
@@ -1,8 +1,3688 @@
 {
   "name": "Agent.Tasks",
   "version": "0.6.0",
-  "lockfileVersion": 1,
```

Comment:
I think this should be rolled back. Can you please use node 10 to build tasks locally?

Reply:
Reverted changes in package-lock.json